src/DIRAC/FrameworkSystem/scripts/dirac_login.py

```diff
@@ -261,9 +264,12 @@ def loginWithCertificate(self):
         """Login with certificate"""
         # Search certificate and key
         if not self.certLoc or not self.keyLoc:
-            cakLoc = Locations.getCertificateAndKeyLocation()
-            if not cakLoc:
-                return S_ERROR("Can't find user certificate and key")
+            if not (cakLoc := getCertificateAndKeyLocation()):
+                problem = "Can't find user certificate and key"
+                if not self.authWith:  # if user do not choose this way
+                    print(HTML(f"<yellow>{problem}</yellow>" + ", trying to connact to DIRAC AS.."))
```

Comment:
```suggestion
                    print(HTML(f"<yellow>{problem}</yellow>" + ", trying to connect to DIRAC AS.."))
```
and do you really need to define a `problem` "variable"?

Reply:
problem "variable" removed